Transformers/HinglishBERTFinetuning.ipynb

```diff
@@ -0,0 +1,978 @@
+{
```

Comment:
<p>I can't believe I am saying this, but this has too many comments for your and my level of expertise. Scrub the extraneous ones, and moves the rest to inline with code instead of giving them separate lines of their own.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>